icepyx/tests/test_Earthdata.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+u"""
```

Comment:
Strings are unicode by default in Python3.

```suggestion
"""
```